packages/cli/lib/util/Configuration.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020-2023 Delft University of Technology and SynTest contributors
+ *
+ * This file is part of SynTest Framework - SynTest Core Graphing Plugin.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Yargs = require("yargs");
+
+export type GeneralOptions = {
+  config: string;
+  applications: string[];
+  plugins: string[];
+  userInterface: string;
+};
+
+export type StorageOptions = {
+  logDirectory: string;
```

Comment:
This should maybe fall under loggingOptions

Reply:
But it is a storage option though.

Reply:
I understand it is an option related to storage. However, as a user, I would look for that option in the logging section. I propose we only put the generic folder options in the storage section.

Reply:
Fixed.